### ISSUE TITLE
Change settings indicators key handling so that one can use arrow keys

### ISSUE THREAD

**Issue by rzhao271** [microsoft/vscode#167470]
Change the key event handling so that the user can tab once into the indicators container, and then use left/right arrows to focus on different indicators.

**Comment by rzhao271** [comment]
1. In the Settings editor, search `editor spaces @lang:yaml`
2. Focus on the `editor.insertSpaces` setting.
3. Pressing tab should focus on the first indicator.
4. Tabbing again should go to the checkbox.
5. Shift-tabbing twice from there should focus the entire setting again.
6. Tab again to the indicator. The left and right arrow keys, along with the home and end keys should allow the user to focus onto the other indicators.
7. The indicators container should keep track of the focused indicator. If the second indicator is focused, and the user tabs to the checkbox and back, the second indicator should be the one focused rather than the first one.
